### ISSUE TITLE
Infinite Scroll demo doesn't work on IE11

### ISSUE THREAD

**Issue by cm325** [s-yadav/angulargrid#62]
The code right out of the infinite scroll demo doesn't work on IE, scrolling down doesn't fire a load. This is a blocker for me at the moment

**Comment by cm325** [comment]
Ok, I could fix this, the scroll handler for IE 11 apparently wants documentElement:

`function scrollHandler() {
              //add support for IE11
              var scrollTop = this.scrollTop || this.scrollY || this.document.documentElement.scrollTop;
              if (options.performantScroll) refreshDomElm(scrollTop);
              if (scope.infiniteScroll) infiniteScroll(scrollTop);
            }`

**Comment by cm325** [comment]
Oh, also, IE 11 also can't do interpolation in style (bug), so you have to use ng-style:

`<div class="img" style="background-image: url({{::shot.imageUrl}});"></div>`

doesn't work, this does:

`<div class="img" ng-style="::{'background-image': 'url('+shot.imageUrl+')'}"></div>`

**Comment by s-yadav** [comment]
I tested it with >ie9 . Things work fine for same demo. Are you trying on mobile devices ?

**Comment by cm325** [comment]
No, not mobile. IE11 on Win10. Does this plunker work for you? It shows at least the ie style interpolation problem- 

http://stackoverflow.com/questions/19014765/angular-bug-ie10-variable-in-style-attribute
